Sprint-3/1-key-implement/3-get-card-value.js

```diff
@@ -25,27 +30,34 @@ function assertEquals(actualOutput, targetOutput) {
 // Given a card string in the format "A♠" (representing a card in blackjack - the last character will always be an emoji for a suit, and all characters before will be a number 2-10, or one letter of J, Q, K, A),
 // When the function getCardValue is called with this card string as input,
 // Then it should return the numerical card value
-const aceofSpades = getCardValue("A♠");
-assertEquals(aceofSpades, 11);
+// const aceofSpades = getCardValue("A♠");
+// assertEquals(aceofSpades, 11);
 
 // Handle Number Cards (2-10):
 // Given a card with a rank between "2" and "9",
 // When the function is called with such a card,
 // Then it should return the numeric value corresponding to the rank (e.g., "5" should return 5).
 const fiveofHearts = getCardValue("5♥");
-// ====> write your test here, and then add a line to pass the test in the function above
+assertEquals(fiveofHearts, 5);
```

Comment:
You handle 10 differently from other numbers - I'd recommend adding a test for 10 as well, to make sure it works.

In general each time you know your code handles some group of input differently, you probably want a test for that example.